Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -963,8 +988,82 @@ def get_mapping_fields_command(service):
     demisto.results(types_map)
 
 
+def get_last_update_in_splunk_time(last_update):
+    """ Transforms the time to the corresponding time on the Splunk server
+
+    Args:
+        last_update: The time to be transformed
+
+    Returns: The corresponding time on the Splunk server
+
+    """
+    last_update_utc_datetime = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
+    dem_params = demisto.params()
+    splunk_timezone = dem_params.get('timezone')
+    if not splunk_timezone:
+        raise Exception('Cannot mirror incidents when timezone is not configured. Please enter the '
+                        'timezone of the Splunk server being used in the integration configuration.')
+    return date_to_timestamp(last_update_utc_datetime + timedelta(minutes=int(splunk_timezone))) / 1000
+
+
+def get_remote_data_command(service):
+    notable_updated_data = []
+    entries = []
+    updated_notable = {}
+    remote_args = GetRemoteDataArgs(demisto.args())
+    last_update_splunk_timestamp = get_last_update_in_splunk_time(remote_args.last_update)
+    notable_id = remote_args.remote_incident_id
+    search = '|`incident_review` ' \
+             '| eval last_modified_timestamp=_time ' \
+             '| where rule_id="{}" ' \
+             '| where last_modified_timestamp>{} ' \
+             '| fields - time ' \
+             '| map search=" search `notable_by_id($rule_id$)`"'.format(notable_id, last_update_splunk_timestamp)
+    demisto.debug('Performing get-remote-data command with query: {}'.format(search))
+
+    for item in results.ResultsReader(service.jobs.oneshot(search)):
+        notable_updated_data.append(parse_notable(item, to_dict=True))
+
+    if notable_updated_data:
+        updated_notable = notable_updated_data[-1]  # a notable can change several times, so taking the latest
+
+    if updated_notable.get('status') == '5' and demisto.params().get('close_incident'):
+        demisto.debug('Closing incident related to notable {}'.format(notable_id))
+        entries.append({
+            'Type': EntryType.NOTE,
+            'Contents': {
+                'dbotIncidentClose': True,
+                'closeReason': 'Notable event was closed on Splunk.'
```

Comment:
isnt there that value from the event that we can take here for close reason? what about close notes?

Reply:
Nope. A user can close a notable on Splunk by given/not giving a comment. If a notable has several comments we cannot differentiate between them. Asked Dominik regarding this and he said to leave it hard-coded.